pkg/agent/agent.go

```diff
@@ -169,8 +173,9 @@ func NewAgent(directiveHandler DirectiveHandler, rolloutsGetterFactory rolloutsG
 		directiveHandler:             directiveHandler,
 		reportRunning:                atomicBool{value: false},
 		agentWatchFieldSelector:      getEnvWithDefault("AGENT_WATCH_FIELD_SELECTOR", "metadata.namespace!=kube-system"),
-		metricsBackoffUntil:          time.Now(),
+		metricsBackoffUntil:          time.Now().Add(defaultMinReportPeriod),
```

Comment:
Just so I understand, was this added to make sure that it doesn't try to report instantly when it starts up and delays the initial report?

Reply:
It was added because otherwise it sends at least 3 requests every 5 seconds, which is too much for ambassador cloud. More-other, we only need to have data every 30s.